### ISSUE TITLE
FNX-5593 ⁃ Refine the "Currently Syncing" screen

### ISSUE THREAD

**Issue by vladikoff** [mozilla-mobile/fenix#2325]
UX: https://mozilla.invisionapp.com/share/QSR2F8UD4PN#/screens/357470305

<img src=https://user-images.githubusercontent.com/128755/57334651-58737600-70ee-11e9-8c71-e6b3819204b3.png width=240 />
cc @grigoryk

**Comment by ekager** [comment]
Whoever ends up working on this ticket, please see #2460 for a11y suggestions. If possible, some of these changing areas should be live regions which read out current syncing status for user feedback.

See @eeejay 's comment:
> Does the user see any kind of label for these different statuses? If so, the same language should be used here. Before you add accessibility announcements, can you try a [live region](https://developer.android.com/reference/android/view/accessibility/AccessibilityNodeInfo.html#setLiveRegion(int)) on a label that is already being used? This gives accessibility services a bit more context and the user some flexibility about how these things are presented.
> 
> If that won't work, an announcement will do.

**Comment by isabelrios** [comment]
This issue is under Ready for QA, although the label is ready for Engineering. I'm confused because either it is not ready for QA yet, since the UI is very different from the screenshot in first comment and it needs some work still or there is actually a bug if it has to be in the ready for QA... @sblatz  what do you think? (not sure who to ask and I say you moved the card :)) 

This is how it looks like now (build from Wed 5/22 1.0.1921):
![Screenshot_20190523-093535](https://user-images.githubusercontent.com/1897507/58234537-12015680-7d3f-11e9-9061-a709f60afaf2.png)

**Comment by sblatz** [comment]
Sorry about that, didn't mean to move this ticket, @isabelrios.

**Comment by vesta0** [comment]
Will be addressed in #289
